content/guides/references/configuration.md

```diff
@@ -594,6 +594,54 @@ See the
 [Custom Dev Server](/guides/component-testing/component-framework-configuration)
 guide for more info.
 
+### Options API
+
+You can also use the `options` API to provide your own project specific
+configuration to your `devServer` for frameworks like angular that do not have a
+`webpack.config` you can pass to cypress.
+
+The `devServer` function receives an `options` argument:
+
+<code-group>
```

Comment:
Should we use `:::cypress-config-example{noJson}` here for consistency?

Reply:
I did not use that because angular only uses typescript so we the js example is not really accurate

Reply:
Oh yeah that makes sense, nevermind 👍🏻